app/util.py

```diff
@@ -119,3 +126,27 @@ def project_user_permissions(form):
         permissions[user] = user_permissions
 
     return permissions
+
+
+# Padding for encryption
+# The lambdas are from StackOverflow, I have no idea how they work
+def pad(e):
```

Comment:
Fair enough. I just wanted to make sure we explored using a library instead of re-inventing the wheel.